src/stoppoints_discovery.rs

```diff
@@ -58,5 +55,29 @@ pub fn stoppoints_discovery(
             }),
             ..Default::default()
         },
-    }))
+    }
+}
+
+pub fn sp_discovery(
+    (actor_addr, query): (State<Addr<Context>>, Query<Params>),
```

Comment:
by curiosity can't this be changed to `actor_addr: Addr<Context>, query: Params` too ?

Reply:
is it done somewhere else? I don’t find how to do that with actix-web

Reply:
I though so, but It's not really important